tests/intern-browserstack.ts

```diff
@@ -0,0 +1,14 @@
+export * from './intern';
```

Comment:
Well the reason I left them as js files was because we load them and extend them. If that is not a pattern we should be using w/ intern4, what's the way we should be doing that? Is it ideal to just have the different json files that we keep in sync?